src/components/ComponentIndexPage/index.js

```diff
@@ -102,11 +112,20 @@ function ComponentIndexPage() {
             .slice()
             .sort(sortBy[activeSortOption])
             .map(({ node }) => {
-              const { name, description, maintainer } = node;
+              const {
+                name,
+                description,
+                maintainer,
+                framework,
+                // eslint-disable-next-line camelcase
+                design_asset,
```

Comment:
```suggestion
                design_asset: designAsset,
```

Could do this to convert from YML to JS and avoid the ESLint issue